bedrock/mozorg/templates/mozorg/about/policy/transparency/jan-jun-2020.html

```diff
@@ -198,7 +198,7 @@ <h3>Trademark</h3>
       </tbody>
     </table>
     <h3>Personal Data Requests</h3>
-    <p>In the Reporting Period, we received 1,058 requests.</p>
+    <p>In the Reporting Period, we received 1,653 requests.</p>
```

Comment:
Reading the comments from @hoosteeno – I think this is where the `*` needs to be placed, rather than on the two numbers there. **We also may need a legal review if the `*` is needed on this number.** 

```suggestion
    <p>In the Reporting Period, we received *1,653 requests.</p>
```